save.go

```diff
@@ -354,14 +354,28 @@ func copyFile(dst, src string) error {
 }
 
 func copyWithoutImportComment(w io.Writer, r io.Reader) error {
-	sc := bufio.NewScanner(r)
-	for sc.Scan() {
-		_, err := w.Write(append(stripImportComment(sc.Bytes()), '\n'))
-		if err != nil {
+	b := bufio.NewReader(r)
+	for {
+		l, err := b.ReadBytes('\n')
+		eof := err == io.EOF
+		if err != nil && err != io.EOF {
 			return err
 		}
+
+		// If we have data then write it out...
+		if len(l) > 0 {
+			// Strip off \n if it exists because stripImportComment
+			_, err := w.Write(append(stripImportComment(bytes.TrimRight(l, "\n")), '\n'))
+			if err != nil {
+				return err
+			}
+		}
+
+		// If we're at the EOF exit
```

Comment:
The comments in this function don't really add anything meaningful. I would delete them.